docs/topics/mta-web-administrator-view.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+
+## Administrator view
+The administrator view is intended to be used by administrators to set up the Tackle instance environment.
+![](/Tackle2/Views/AdminMenu.png)
+
+### Credentials
+This management module enables administrators to create and manage credentials for access to private repositories.  It also allows for the architects to assign the credentials to applications without knowing their contents.
```

Comment:
Credentials are not assigned to applications in the Credentials view, but in the Application Inventory view.